syntax = "proto3";

package grpc_performance;

service DataStreamer {
  rpc streamData (StreamRequest) returns (stream SingleMessage) {}
}

message StreamRequest {
  bytes jsonRequest = 1;
}

message SingleMessage {
  bytes message = 1;
}
